collection/sites/googledocs.yml

```diff
@@ -0,0 +1,11 @@
+name: All Google Documents (disable Google Sheets before enable it!)
+host: docs.google.com
+example:
+    - 'https://docs.google.com/spreadsheets/d/1f988o68HDvk335xXllJD16vxLBuRcmm3vg6U9lVaYpA'
+    - 'https://docs.google.com/spreadsheets/u/1/d/e/2PACX-1vQ7iYECa_ziPWygKBf9Og-PkQl5-gdnFRU-jP-z1FyjLfLXMAdRJN1J5Lx31NjnCq5lkRcL-T0Idh4_/pubhtml'
+    - 'https://docs.google.com/spreadsheets/d/e/2PACX-1vSbbciMU7t5dCB8auk-nyK-p-25TJjgUrzBuMCAiq_2ZDqoMSFAlZCctIC7y3TXS71ys_MRdoAU4Jqr/pubchart?oid=1729348495'
+extract: "!docs\\.google\\.com/(?'id'(document|spreadsheets|presentation|forms)(((/\\d)+|(/\\w)+)+)/([^/?]+)(/[^/]+)?)!"
+iframe:
+    height: 500
+    width: '100%'
```

Comment:
It should be consistent, so quoted both of `width` and `height` values for example.

Reply:
Fixed